models/migrations/v1_19/v242.go

```diff
@@ -0,0 +1,25 @@
+// Copyright 2023 The Gitea Authors. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+package v1_19 //nolint
+
+import (
+	"code.gitea.io/gitea/modules/setting"
+	"xorm.io/xorm"
+)
+
+// AlterPublicGPGImportKeyContentFieldToMediumText: set GPGImportKey Content field to MEDIUMTEXT
+func AlterPublicGPGImportKeyContentFieldToMediumText(x *xorm.Engine) error {
+	sess := x.NewSession()
+	defer sess.Close()
```

Comment:
Aren't the migrations run already in separate transactions?

Reply:
It's not, because the parameter is `x` not a `session`.